README.md

```diff
@@ -151,7 +152,7 @@ release:
 
 #### With Yarn 2 / Plug'n'Play
 
-If you are using [Yarn Plug'n'Play](https://yarnpkg.com/features/pnp), you should use a custom `version` command so that the action can resolve the `changeset` CLI:
+If you are using [Yarn Plug'n'Play](https://yarnpkg.com/features/pnp), you should use a custom `version` command so that the action can resolve the `changeset` CLI.
```

Comment:
I think `:` is correct here, the following codes are used for explanation.

Reply:
Done